app/javascript/app/components/intro/intro-component.jsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { themr } from 'react-css-themr';
+import Button from 'components/button';
 import cx from 'classnames';
 
 import styles from './intro-styles.scss';
 
 const Intro = props => {
-  const { title, description, theme, textColumns } = props;
+  const { title, description, theme, textColumns, button } = props;
   return (
     <div className={theme.intro}>
-      <h2 className={theme.title}>{title}</h2>
-      <p
-        className={cx(theme.description, textColumns ? theme.columns : '')}
-        dangerouslySetInnerHTML={{ __html: description }} // eslint-disable-line
-      />
+      <div className={theme.main}>
+        <h2 className={theme.title}>{title}</h2>
+        <p
+          className={cx(theme.description, textColumns ? theme.columns : '')}
+          dangerouslySetInnerHTML={{ __html: description }} // eslint-disable-line
+        />
+      </div>
+      {button && (
```

Comment:
It might have been easier to handle this as children, but I dont think it matters now. Good stuff.